src/config/ConfigManager.hpp

```diff
@@ -11,7 +11,7 @@ class CConfigManager {
     void parse();
 
     std::deque<std::string> m_dRequestedPreloads;
-    std::string getMainConfigPath();
+    static std::string getMainConfigPath();
```

Comment:
whats the point

Reply:
Clangd suggestion.

Reply:
haven't I told you how much I care about what clangd says